### PR TITLE
Reword users

### PR DESCRIPTION
**Pull request by knarrff**

### CODE REVIEW

File: paper_teaching-learning-RSE.md [hunk 1]
```diff
@@ -215,10 +215,12 @@ The RSE should be able to interact with the repositories of this specific domain
 
 
 #### Outside Party interaction (USERS)
-While in a traditional SE context, you might get away with not interacting with people outside your project.
-But in a research context, this will certainly be the case and involves users, other developers,  upto funders.
-Additionally, this is oftentimes a two-way interaction with RSEs in a specific domain learning new findings, techniques, algorithms, 
-etc. to be able to implement software that is up-to-date with the body of knowledge of that domain.
+Research software is often developed as part of the research process itself, and like the research process, it will change often and not unlikely in unpredictible ways.
+Because of this, research software often has to be developed very closely to its users, the researchers.
```
Comment (CaptainSifff) [inline paper_teaching-learning-RSE.md:219]: In an attempt of shortening:
```suggestion
Research software is often developed as part of the research process itself, therefore it often has to be developed very closely to its users, the researchers.
```

**Review by CaptainSifff — CHANGES_REQUESTED** [review]

File: paper_teaching-learning-RSE.md [hunk 2]
```diff
@@ -215,10 +215,12 @@ The RSE should be able to interact with the repositories of this specific domain
 
 
 #### Outside Party interaction (USERS)
-While in a traditional SE context, you might get away with not interacting with people outside your project.
-But in a research context, this will certainly be the case and involves users, other developers,  upto funders.
-Additionally, this is oftentimes a two-way interaction with RSEs in a specific domain learning new findings, techniques, algorithms, 
-etc. to be able to implement software that is up-to-date with the body of knowledge of that domain.
+Research software is often developed as part of the research process itself, and like the research process, it will change often and not unlikely in unpredictible ways.
+Research software is often developed as part of the research process itself, therefore it often has to be developed very closely to its users, the researchers.
```
Comment (CaptainSifff) [inline paper_teaching-learning-RSE.md:219]: Sorry, to bother again, but we caught a duplication somehow here.

**Review by CaptainSifff — CHANGES_REQUESTED** [review]

File: paper_teaching-learning-RSE.md [hunk 3]
```diff
@@ -215,10 +215,12 @@ The RSE should be able to interact with the repositories of this specific domain
...
+This often means that it is hard to distinguish roles like developers and users because people often represent multiple roles.
```
Comment (jpthiele) [inline paper_teaching-learning-RSE.md:220]: There are too many 'often' in these two sentences, maybe we can invert the second one, e.g.
'Then, roles like developers and users can seldom be distinguished as most people represent multiple roles.'

**Comment by CaptainSifff** [comment]
any updates, here?

**Comment by CaptainSifff** [comment]
See #70

**Pull request merged**